templates/CRM/UF/Form/Group.tpl

```diff
@@ -47,10 +47,12 @@
 {else}
     <table class="form-layout">
       {foreach from=$entityFields item=fieldSpec}
-        {assign var=fieldName value=$fieldSpec.name}
-        <tr class="crm-{$entityInClassFormat}-form-block-{$fieldName}">
-          {include file="CRM/Core/Form/Field.tpl"}
-        </tr>
+        {if not in_array($fieldSpec.name, $advancedFieldsConverted)}
```

Comment:
This ensures that the new fields don't show on the main part of the form as well as on the advanced settings section

Reply:
not hugely important but I think !in_array would be more consistent with elsewhere than 'not'